universe/marathon.json.mustache

```diff
@@ -48,7 +48,10 @@
   "MARATHON_SINGLE_INSTANCE_APP":"true",
   "DCOS_SERVICE_NAME": "{{service.name}}",
   "DCOS_SERVICE_PORT_INDEX": "0",
-  "DCOS_SERVICE_SCHEME": "http"
+  "DCOS_SERVICE_SCHEME": "http",
+  "HAPROXY_0_HTTP_BACKEND_PROXYPASS_PATH": "/{{service.name}}",
+  "HAPROXY_0_PATH": "/{{service.name}}/v1/datacenter",
+  "HAPROXY_0_VHOST": "{{service.vhost}}"
 },
```

Comment:
{{{service.vhost}}}

Reply:
no its only {{service.vhost}}
all the params read like this ex. "DCOS_SERVICE_NAME": "{{service.name}}",

Reply:
using {{{}}}, mustache will not escape html chars("&<> etc). Otherwise it escapes html chars.

Reply:
make scenes, it is getting using like {{#service.external_data_center_urls}} , 
so which one should be used ?? 
{{#}} or {{{}}}  ?

Reply:
{{{}}} used this till now in other serivces. Not sure about other syntax

Reply:
ok changed with working fine with both {{}} and {{{}}}. 

now changed with {{{}}}